PennyPincherExample/PennyPincherExample/ViewController.swift

```diff
@@ -52,8 +49,61 @@ class ViewController: UIViewController, UITextFieldDelegate {
         recognizerResultLabel.text = "Template: \(template.id), Similarity: \(similarityString)"
     }
    
+    //MARK:- Actions
+    @IBAction func didTapAddTemplate(_ sender: AnyObject) {
+        if
+            let text = templateTextField.text,
+            let template = PennyPincher.createTemplate(text, points: gestureView.points) {
```

Comment:
Yes, that's existing code from the project. I just moved all the actions together, and just added some code to this one. What if I replace the if let with guard?